source/includes/usage-examples/code-snippets/distinct.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func main() {
+	var uri string
```

Comment:
`uri` should be a const above main, per the [Go Driver Code Example Patterns and Style Reference](https://docs.google.com/document/d/1FtUKxOaEmdFf4d3HcXAVIGed2awuwIWdkWc0YQIwPvk/edit#heading=h.afeu4uin7cta) document.

Reply:
Thank you for updating the patterns doc. I now see the pattern is to use the `MONGODB_URI` environment variable.